spec/dummy/package.json

```diff
@@ -2,14 +2,13 @@
   "name": "dummy",
   "private": true,
   "dependencies": {
-    "@rails/actioncable": "^6.0.0-alpha",
-    "@rails/activestorage": "^6.0.0-alpha",
-    "@rails/ujs": "^6.0.0-alpha",
-    "@rails/webpacker": "^4.0.7",
+    "@hotwired/turbo-rails": "^8.0.20",
+    "@rails/actioncable": "^6.1.710",
+    "@rails/activestorage": "^6.1.710",
+    "@rails/webpacker": "^5.4.4",
     "rails-erb-loader": "^5.5.2",
-    "turbolinks": "^5.2.0",
-    "webpack": "^4.0.0",
-    "node-sass": "^4.14.1"
+    "sass-embedded": "^1.79.4",
```

Comment:
Think `sass` might be more universally compatible (I've heard of cross compilation with sass-embedded issues on macos?) - and performance benefit is negligible in specs

Reply:
Totally agree - but `sass` is a separate package that relies on the standard dart-sass implementation. `sass-embedded` is a recompilation of dart-sass into js.

Reply:
changed to sass